contributors/devel/e2e-tests.md

```diff
@@ -145,7 +145,13 @@ go run hack/e2e.go -- -v --down
   This will not do anything besides allow you to verify that the go code compiles.
 
 * If you want to run your e2e testing framework without re-provisioning the e2e setup,
-  you can do so via `make WHAT=test/e2e/e2e.test` and then re-running the ginkgo tests.
+  you can do so via `make WHAT=test/e2e/e2e.test`
+  
+* You can then create a symlink pointing `e2e.test` to `_output/local/bin/linux/amd64/e2e.test`, 
+  and re-run the ginkgo tests by providing relevant flags. For example:
+```sh
+e2e.test --host="127.0.0.1:8080" --provider="local" --repo-root="path/to/main/repo" --kubeconfig="path/to/.kube/config" --ginkgo.focus="regex"
```

Comment:
This is a bit hacky. I found a more canonical way to run e2e tests on a local cluster:
```sh
# Run tests matching the regex "\[Feature:Performance\]" on a local cluster (specified with --provider)
go run hack/e2e.go -- -v --test --test_args="--ginkgo.focus=\[Feature:Performance\]" --provider=local because it's more 
# Alternatively specifying it with KUBERNETES_PROVIDER; I personally prefer --provider because it's more discoverable than env 
KUBERNETES_PROVIDER=local go run hack/e2e.go -- -v --build --up --test --down
```

I just took another look at this doc. This section talks about "Building the Tests" whereas the above section talks about "Building Kubernetes and Running the Test" -- which is somewhat redundant. 

Please, move this section to under "Building Kubernetes and Running the Test", add a section for "Running the test"and move the script for running tests locally into the example of running e2e tests. 

You can find supported flags/options with `go run hack/e2e.go -- --help`. In this doc, `go run hack/e2e.go --help` is mentioned, but is not correct. Please correct it also.